### ISSUE TITLE
Import model error

### ISSUE THREAD

**Issue by LeoLee3d** [TheGameCreators/GameGuruRepo#3716]
Hi everyone!
I can't use Import Model. Every time I want to load a model it gives me an error:

"vconv.exe application error - App could not be initialized. (0xc000007b)"

I've already reloaded GG and reboot the computer, but the problem persists.
I also verified it on Steam and two validation errors occurred.

Thanks for any help.

**Comment by MonkeyFrogStudio** [comment]
Is this with GameGuru Classic (you mentioned GG, so I am assuming yes) or with GameGuru MAX? Also, what is models file format type - obj, fbx, etc.? Any information you can provide will help the developers to better help you.

**Comment by LeoLee3d** [comment]
Gameguru Classic, fbx format.
I don't know where I can upload an image.

**Comment by MonkeyFrogStudio** [comment]
You can just drag-n-drop the image right into the text window here and it will be added.

**Comment by LeoLee3d** [comment]
![GG-importer-error](https://user-images.githubusercontent.com/113857664/190926865-ffd2fb34-68a2-4b50-bc14-4a2ae616cc98.jpg)

**Comment by Kitakazi** [comment]
Are you on 64 bit windows?

**Comment by LeoLee3d** [comment]
Yes.
Everything  in GameGuru works fine, only the model importer doesn't.

Em qua., 21 de set. de 2022 às 15:22, David Brandt ***@***.***>
escreveu:

**Comment by davetgc** [comment]
Can you please post the model you're trying to import? Or, it you can't share it, you can email it privately to Lee@thegamecreators.com, with a link to this issue.

**Comment by LeoLee3d** [comment]
Any FBX model I try to import has the same problem.
Only X models works fine.

Em qui., 22 de set. de 2022 às 03:04, davetgc ***@***.***>
escreveu:

**Comment by LeeBamberTGC** [comment]
Can you please post the model you're trying to import?

**Comment by Jackdanielsvs68** [comment]
I have a 32-bit version after reinstalling Windows, because it was installed in the steamgame/steamapps folder, but steam reinstalled it in the steamlibrary folder, but I can use both. The 32-bit version easily imports the characters, which I create with makehuman, animate with mixamo, load with importer and concatenate with ximporter. I get a runtime error with the 64 bit version. The previous file is always kept in the importer/temp folder, and the importer uses it for the next one, so it will be a big mess. But even if I delete it every time, it's not good either. I'm using apbr.anim.fx.

**Comment by LeeBamberTGC** [comment]
@LeoLee3d All my test FBX models import as expected at my end. Alas, I will need one of your FBX models to see the crash.

**Comment by Jackdanielsvs68** [comment]
![Kéf142](https://user-images.githubusercontent.com/94233170/209564635-90f87cf9-c604-476a-bc13-8ef718cd8a44.png)
![Képernyőfelvétel (139)](https://user-images.githubusercontent.com/94233170/209564639-798beb38-0435-440d-8c09-e508fc849692.png)
![Képernyőfelvétel (141)](https://user-images.githubusercontent.com/94233170/209564642-6ad0da02-372c-47ff-b9c8-3b3b03a3777a.png)
![Képernyőfelvétel (143)](https://user-images.githubusercontent.com/94233170/209564649-c3e46456-89bb-4c86-90b7-2f1ec600ba13.png)
![Képernyőfelvétel (144)](https://user-images.githubusercontent.com/94233170/209564655-4f096f72-8614-4cf2-884e-df4855647cb1.png)
![Képernyőfelvétel (145)](https://user-images.githubusercontent.com/94233170/209564661-1416872a-2b1f-4539-b07b-17ff9d4d34a8.png)
![Képernyőfelvétel (146)](https://user-images.githubusercontent.com/94233170/209564663-bcbcb8b2-1a91-4bdd-a193-e5ead08c86f0.png)
![Képernyőfelvétel (147)](https://user-images.githubusercontent.com/94233170/209564666-d813fbe8-0e2c-4a59-89bd-4d896643017f.png)
![Képernyőfelvétel (148)](https://user-images.githubusercontent.com/94233170/209564671-48f65342-1e92-490d-a66b-a64fae5b1639.png)

**Comment by Jackdanielsvs68** [comment]
I already know where the error is and I can handle it, but there may be many people who think that they must be the ones who screw something up. The 64 bit version of GG is constantly updated via steam, the 32 bit version is not. What is on it, I copied. Currently, I prefer to use it to import models, because the other one is very complicated that way.

**Comment by LeoLee3d** [comment]
I can import right now, but before import, I have to delete the temporary
dbo file every time I use the importer.

@Lee Bamber - Here's the fbx:
https://drive.google.com/file/d/1bRnmzceqqmUHWcxsefJXkioiBZlFVGic/view?usp=share_link

Em seg., 26 de dez. de 2022 às 12:48, Jackdanielsvs68 <
***@***.***> escreveu:

**Comment by Jackdanielsvs68** [comment]
Yes, that is. But with the old 32-bit version, you don't need to delete the temporary dbo, the importer does it, and anyone who doesn't know this will get an error. In addition, it is quite annoying that if I import four or five characters, each with 6 animations.

**Comment by LeeBamberTGC** [comment]
@LeoLee3d I cannot download your link, it says the file is in the owner's bin.  

@JackDanielsv268 Can you send me these files shown in your shots. Once I have the ZIP with model and all associated files that show the above issue, I can trace which files are left in the temp folder and ensure they are deleted in the 64 bit version.

**Comment by LeoLee3d** [comment]
I don't have that file any more and I don't remember what the file is. :)
But no problem, I'll wait for the update.
Thanks!

Em dom., 14 de mai. de 2023 às 16:56, Lee Bamber ***@***.***>
escreveu:

**Comment by LeoLee3d** [comment]
Awesome, thanks!

Em seg., 15 de mai. de 2023 às 05:32, Lee Bamber ***@***.***>
escreveu:

**Comment by Jackdanielsvs68** [comment]
Hello! I think I'll wait until June to try. I'm halfway through a game and I don't want it to be damaged by a bug. If I've solved it so far, I can handle that little time. :) But thank you very much for your efforts, I will definitely let you know.

**Comment by LeeBamberTGC** [comment]
@Jackdanielsvs68 June update is out and you are right not jeopardise your current project by swapping out your engine for an update :)  When you get the chance to confirm the above, feedback here, in the meantime I will close this as resolved for now.

**Comment by Jackdanielsvs68** [comment]
I tried it, it works fine, the temp file is replaced on the next import. Thanks for the correction.
